src/core/dfn.js

```diff
@@ -17,6 +17,19 @@ export function run() {
       dfn.dataset.dfnType = "idl";
     }
 
+    // per https://tabatkins.github.io/bikeshed/#dfn-export
+    // a dfn typed with something else than dfn
+    // and not marked with noexport is to be exported
+    // We also skip "imported" definitions via data-cite
+    if (
+      dfn.dataset.dfnType &&
+      dfn.dataset.dfnType !== "dfn" &&
+      !dfn.dataset.cite &&
+      !dfn.dataset.noexport
```

Comment:
This might be `noExport` in ReSpec. I'll try to verify when I'm on a computer.

Rest looks good though.

Reply:
```suggestion
      !dfn.dataset.noExport
```
per https://github.com/w3c/respec/blob/develop/src/core/data-cite.js#L105
(although `data-noExport` doesn't seem to be documented in https://github.com/w3c/respec/wiki)